libs/sdk-backend-tiger/src/backend/tigerSpecificFunctions.ts

```diff
@@ -26,6 +26,8 @@ import {
     DeclarativeTables,
     DeclarativeAnalytics,
     JsonApiWorkspaceInDocument,
+    DependentEntitiesRequest,
+    DependentEntitiesResponse,
```

Comment:
Please, reexport these so you can import them from SPI package and not API package.

Reply:
fixed.